packages/neo-one-website/src/components/content/Sidebar.tsx

```diff
@@ -27,6 +27,80 @@ const MobileWrapper = styled(Box)`
   }
 `;
 
+const fadeSlideIn = keyframes`
+  0% {
+    transform: translateY(40px);
+    opacity: 0
+    }
+  10% {
+    opacity: .1
+  }
+  20% {
+    opacity: .2
+  }
+  30% {
+    opacity: .3
+  }
+  40% {
+    opacity: .4
+  }
+  50% {
+    opacity: .5
+  }
+  60% {
+    opacity: .6
+  }
+  70% {
+    opacity: .7
+  }
+  80% {
+    opacity: .8
+  }
+  90% {
+    opacity: .9
+  }
+  100% {
+    opacity: 1
+  }
+`;
+
+const fadeSlideOut = keyframes`
+  0% {
+    opacity: 1
+    }
+  10% {
+    opacity: .9
+  }
+  20% {
+    opacity: .8
+  }
+  30% {
+    opacity: .7
+  }
+  40% {
+    opacity: .6
+  }
+  50% {
```

Comment:
I don't think you need to explicitly specify every keyframe. In this case you should be able to just specify the start and end opacity.